libbeat/reader/readfile/line.go

```diff
@@ -67,6 +67,7 @@ func NewLineReader(input io.Reader, config Config) (*LineReader, error) {
 		decodedNl:  terminator,
 		inBuffer:   streambuf.New(nil),
 		outBuffer:  streambuf.New(nil),
+		logger:     logp.NewLogger("readfile_line"),
```

Comment:
Nit: can we call this `reader_readfile_line` for consistency? All the other reader log selectors start with `reader_`.

Reply:
Or just `reader_line` is fine too.

Reply:
I like `reader_line` better 👍